### ISSUE TITLE
Component: Table inside Panel

### ISSUE THREAD

**Issue by mazerdeci** [primefaces/primeng#16986]
### Describe the bug

If you put a table component inside a panel component, panel and table footer contents mixed up.
Panel footer area shows table's footer.
Please check stackblitz example.
Workaround; If you change #footer of the table component (new syntax) to pTemplate="footer" (old syntax) it works fine.

### Environment

Windows 11

### Reproducer

https://stackblitz.com/edit/github-jvgew3he-ncnqccfv

### Angular version

18.2.13

### PrimeNG version

18

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v22.11.0

### Steps to reproduce the behavior

Please check stackblitz example

### Expected behavior

Panel Footer should be displayed correctly

**Comment by vanackej-onemw** [comment]
Hi.

There is a global problem in this release regarding the templates. In all components.
There is an issue raised on the panel component, I also found that it affects the inplace component and the select component.

In the select component, the selectedItem template has no effect.

Partial exemple from my codebase : 

```html
    <p-select inputId="resolutionInput" [formControl]="pointsQueryForm.controls.resolution"
              [options]="resolutionOptions">
      <ng-template pTemplate="selectedItem">
        <span>
          I can write anything here it will be ignored
        </span>
      </ng-template>
      <ng-template let-option pTemplate="item">
        <span>
          {{ ("simulation.results.resolution." + option) | transloco }}
        </span>
      </ng-template>
    </p-select>
```

It is quite disappointing that a massive bug affecting basically all the components of this library is introduced just in the official release version (I was using the latest Release candidate without any of those template problems)

I assume that if the problem is global, the fix can be too, no ?

I didn't change the Angular version when upgrading to PrimeNG 18 release, so the issue comes from the library codebase.

I find this commit suspicious, as it basically refactor the whole templating processing a few days ago, but didn't have time to check if it's the reason or not : https://github.com/primefaces/primeng/commit/295e8662ffaad5abcf88bf4732474fa55438be42

@mertsincan I saw you were assigning other issues related to this problem on this release, so tagging you to catch your attention here.

Thanks for your hard work, hope this global templating issue can be fixed soon.
